pkg/manager/impl/execution_manager.go

```diff
@@ -1194,6 +1194,9 @@ func (m *ExecutionManager) CreateWorkflowEvent(ctx context.Context, request admi
 		return nil, errors.NewFlyteAdminErrorf(codes.FailedPrecondition,
 			"Cannot go from %s to %s for workflow execution %v",
 			wfExecPhase.String(), request.Event.Phase.String(), request.Event.ExecutionId)
+	} else if wfExecPhase == core.WorkflowExecution_ABORTING && !common.IsExecutionTerminal(request.Event.Phase) {
+		return nil, errors.NewFlyteAdminErrorf(codes.FailedPrecondition,
+			"Invalid phase change from aborting to %s for workflow execution %v", request.Event.Phase.String(), request.Event.ExecutionId)
```

Comment:
Is there a possibility of receiving the ABORTING phase again for the same workflow execution.
If thats the case, do we want to flag that as error of invalid phase change.

Reply:
it could perhaps be possible but the first block of this if statement already checks that we don't receive the same phase first